packages/lib-ethers/utils/deploy.ts

```diff
@@ -11,14 +11,12 @@ import {
 interface IOwners {
   feePool: string
   bammOwner: string
-  legalEntity: string
   governorBravo: string
 }
 
 const contractOwners: IOwners = {
   feePool: "0x7095F0B91A1010c11820B4E263927835A4CF52c9",
```

Comment:
This seems like a Liquity contract (https://etherscan.io/address/0x7095F0B91A1010c11820B4E263927835A4CF52c9). Is that correct?

Reply:
It's the feePool contract address provided by B. protocol, the same one used in Liquity BAMM contracts, more info can be seen here: https://docs.bprotocol.org/technical-documentation/smart-contracts/liquity and here: https://discord.com/channels/866378471868727316/918241759983710298/1156303650222768188